dev/travis/config/apache_virtual_host

```diff
@@ -0,0 +1,43 @@
+<VirtualHost *:80>
+    DocumentRoot %MAGENTO_ROOT%
+    ServerName %MAGENTO_HOST_NAME%
+
+    <Directory "%MAGENTO_ROOT%">
+        Options FollowSymLinks MultiViews ExecCGI
+        AllowOverride All
+        Require all granted
+    </Directory>
+
+    <IfModule mod_fastcgi.c>
+        AddHandler php7-fcgi .php
+        Action php7-fcgi /php7-fcgi
+        Alias /php7-fcgi /usr/lib/cgi-bin/php7-fcgi
+        FastCgiExternalServer /usr/lib/cgi-bin/php7-fcgi -host 127.0.0.1:9000 -pass-header Authorization
+        <Directory /usr/lib/cgi-bin>
+            Require all granted
+        </Directory>
+    </IfModule>
+</VirtualHost>
+<VirtualHost *:443>
+    DocumentRoot %MAGENTO_ROOT%
+    ServerName %MAGENTO_HOST_NAME%
+    SSLEngine on
+    SSLCertificateFile %TRAVIS_BUILD_DIR%/%MAGENTO_HOST_NAME%.cert
+    SSLCertificateKeyFile %TRAVIS_BUILD_DIR%/%MAGENTO_HOST_NAME%.key
+
+    <Directory "%MAGENTO_ROOT%">
+        Options FollowSymLinks MultiViews ExecCGI
+        AllowOverride All
+        Require all granted
+    </Directory>
+
+    <IfModule mod_fastcgi.c>
```

Comment:
Why do we need fastcgi?

Reply:
I just copied the travis virtual host config from the main magento2 repo, but expanded it to add a virtual host for use over HTTPS  https://github.com/magento/magento2/blob/2.3-develop/dev/travis/config/apache_virtual_host

Reply:
As for "why do we need fastci?" - I'll be honest, I have no idea! If there's a better option I am all ears 😄

Reply:
I think this block can be removed, am I wrong?

Reply:
Will test it now!

Reply:
Confirmed that without this module, PHP does not get processed and HTTP(s) requests  to the server return raw PHP code instead of HTML. So, I think it is necessary, at least in this current setup 🤷‍♂ I am not an expert in Apache or PHP and don't know what a "typical" PHP setup looks like so I don't trust  myself in this question nor should anyone else trust me!